_posts/2020-09-15-data-sharing-at-scale.md

```diff
@@ -0,0 +1,301 @@
+      Policies:
+        - PolicyName: 'root'
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action: 
+                  - 'glue:*'
+                Resource: '*'
+              - Effect: Allow
+                Action:
+                  - 'logs:CreateLogGroup'
+                  - 'logs:CreateLogStream'
+                  - 'logs:PutLogEvents'
+                  - 'logs:AssociateKmsKey'
+                Resource: '*'
+              - Effect: Allow
+                Action: 's3:ListBucket'
+                Resource: !GetAtt MySourceDataStore.Arn
+              - Effect: Allow
+                Action: 's3:GetObject'
+                Resource: !Sub
+                  - '${Bucket}/*'
+                  - { Bucket: !GetAtt MySourceDataStore.Arn }
+
+  MySourceCrawler:
+    Type: AWS::Glue::Crawler
+    Properties:
+      Name: my-source-data-crawler
+      Role: !GetAtt MySourceCrawlerRole.Arn
+      DatabaseName: !Ref MySourceGlueDatabase
+      Targets:
+        S3Targets:
+          - Path: !Ref MySourceDataStore
+      SchemaChangePolicy:
+        UpdateBehavior: 'UPDATE_IN_DATABASE'
+        DeleteBehavior: 'LOG'
+
+  SourceCrawlerLakeGrants:
+    Type: AWS::LakeFormation::Permissions
+    Properties:
+      DataLakePrincipal:
+        DataLakePrincipalIdentifier: !GetAtt MySourceCrawlerRole.Arn
+      Permissions:
+        - ALTER
+        - DROP
+        - CREATE_TABLE
+      Resource:
+        DatabaseResource:
+          Name: !Ref MySourceGlueDatabase
+
+  DatalakeLocation:
+    Type: AWS::LakeFormation::Resource
+    Properties: 
+      ResourceArn: !GetAtt  MySourceDataStore.Arn
+      RoleArn: !Sub arn:aws:iam::${AWS::AccountId}:role/aws-service-role/lakeformation.amazonaws.com/AWSServiceRoleForLakeFormationDataAccess
+      UseServiceLinkedRole: true
+```
+
+Add the above resources to your datalake.yml file and run an update stack command.
+
+```sh
+aws cloudformation update-stack --stack-name my-source-glue-stack --template-body file://datalake.yml --capabilities "CAPABILITY_NAMED_IAM"
+```
+
+When you have deployed the crawler, you should be able to see and run it in the Glue Console. 
+
+<center><img src="/img/lake-formation/04.png" /></center><br/>
+
+Click on run and wait for your crawler to be completed.
+
+Once your crawler runs successfully, you can go to tables menu item and see your table being generated.
+
+<center><img src="/img/lake-formation/05.png" /></center><br/>
+
+## Cross-account grant
+
+Once you have your Glue database and tables created, you will be able to manage the database access via Lake Formation. 
+
+To enable cross-account access, you will need to add a Lake Formation grant and specify the consumer account number.
+
+```yml
+ CrossAccountLakeGrants:
+    Type: AWS::LakeFormation::Permissions
+    Properties:
+      DataLakePrincipal:
+        DataLakePrincipalIdentifier: "300000000015" // Consumer account number
```

Comment:
super minor but I think to add a comment in YAML you have to use a `#` instead of a `//`